app/components/icons/defaultAvatar.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
rename fine
all .tsx files should start with capital letter.
So  the name should be DefaultAvatarIcon